ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/RecoveryScanTest.java

```diff
@@ -66,7 +66,8 @@ public static void beforeClass() throws Exception {
 
         manager = RecoveryManager.manager(RecoveryManager.DIRECT_MANAGEMENT);
 
-        manager.addModule(new XARecoveryModule()); // we only need to test the XARecoveryModule
```

Comment:
Same comment here (the change is unrelated to the issue)

Reply:
Changes are related

Reply:
Thanks